pkg/portal/portal.go

```diff
@@ -261,21 +265,40 @@ func (p *portal) unauthenticatedRoutes(r *mux.Router) {
 }
 
 func (p *portal) aadAuthenticatedRoutes(r *mux.Router) {
-	for _, name := range AssetNames() {
-		if name == "v1/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
-			continue
-		}
+	var names []string
 
-		if name == "v2/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/v2").HandlerFunc(p.indexV2)
-			continue
+	err := filepath.Walk(".", func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			log.Fatal(err)
+			return nil
 		}
 
-		fmtName := strings.TrimPrefix(name, "v1/build/")
-		fmtName = strings.TrimPrefix(fmtName, "v2/build/")
+		names = append(names, path)
+		return nil
+	})
 
-		r.NewRoute().Methods(http.MethodGet).Path("/" + fmtName).HandlerFunc(p.serve(name))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, name := range names {
+		regexp := regexp.MustCompile(`v[1,2]/build/.*\..*`)
+		name := regexp.FindString(name)
+		if len(name) > 0 {
+			switch name {
+			case "v1/build/index.html":
+				r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
+			case "v2/build/index.html":
+				r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.indexV2)
+			case "":
+				break
+			default:
+				fmtName := strings.TrimPrefix(name, "v1/build/")
+				fmtName = strings.TrimPrefix(fmtName, "v2/build/")
+
+				r.NewRoute().Methods(http.MethodGet).Path("/" + fmtName).HandlerFunc(p.serve(name))
+			}
+		}
```

Comment:
you don't need to check `len(name)>0` because you have the `""` case in the switch, this is why I suggested it